app/views/taxons/_tagged_content_list.html.erb

```diff
@@ -3,30 +3,20 @@
 <% if presented_taxon.tagged_content.any? || !is_grid %>
   <div class="grid-row">
     <div class="column-two-thirds">
-      <div class="parent-topic-contents">
-        <div class="topic-content">
-          <% if presented_taxon.child_taxons.present? %>
-            <h2><%= presented_taxon.title %> guidance</h2>
-            <p><%= presented_taxon.description %></p>
-          <% end %>
-
-          <ol>
-            <% presented_taxon.tagged_content.each_with_index do |content_item, index| %>
-              <li>
-                <h2>
-                  <%= link_to(
-                    content_item.title,
-                    content_item.base_path,
-                    data: is_grid ? presented_taxon.options_for_tagged_content(index: index) :
-                              presented_taxon.options_for_leaf_content(index: index)
-                  ) %>
-                </h2>
-                <p><%= content_item.description %></p>
-              </li>
-            <% end %>
-          </ol>
+      <% if presented_taxon.child_taxons.present? %>
+        <div class="topic-header">
+          <h2><%= presented_taxon.title %> guidance</h2>
+          <p><%= presented_taxon.description %></p>
         </div>
-      </div>
+      <% end %>
+
+      <% data_options_type = is_grid ? :options_for_tagged_content : :options_for_leaf_content %>
+      <% heading_level = heading_level ? heading_level : false %>
+      <%= render 'components/taxon-list',
+        {
+          heading_level: heading_level,
+          items: taxon_list_params(presented_taxon, data_options_type: data_options_type)
+        } %>
```

Comment:
Minor: inconsistent formatting for the render statement, between here and accordion.html.erb.  Not sure which is best. I kind of like the way of doing it that was removed from accordion, it takes up less space.